scripts/chroot.sh

```diff
@@ -40,13 +40,6 @@ apt-get install -f
 apt-get -qq -y install xorg sddm
 apt-get -qq -y install lxqt
 
-#  may fix the black screen issue 
-apt-get -qq -y install xserver-xorg-video-intel
-
-# or alternative
-apt-get -qq -y install openbox pcmanfm-qt lxqt-admin lxqt-common lxqt-config lxqt-globalkeys lxqt-notificationd lxqt-panel lxqt-policykit lxqt-powermanagement lxqt-qtplugin lxqt-runner lxqt-session lxqt-sudo
```

Comment:
We are using plasma now, not LXQT, right?

Reply:
sddm is the one for plasma and lxqt and it has known issues with password free login. I tried to install lubuntu-desktop along to get a login and see what is happening, That failed so far.

Idea, maybe we should create a default user in the chroot with addpwd. I have no idea how hotelos as a user gets generated.